common/locales/en-US/main.ftl

```diff
@@ -140,6 +140,7 @@ friends = Friends
     .already-friends = Already Friends
     .nothing-to-see-here = Nothing to see here
     .share = Share DID
+    .copy-did = Copy DID
```

Comment:
This should exist already in Dev, we can instead of including a button wrap the component in a context menu with the following items: 
![image](https://github.com/Satellite-im/Uplink/assets/1770198/6fca87a6-ea7d-4ff2-a070-f757679cba84)
 Please use the same icons and translations for consistency